lib/game.rb

```diff
@@ -25,45 +25,48 @@ def initialize(game_io: ConsoleIO.new, board_size:, player_2:, board_presenter:
 
     def play
         while status == :play
-            take_turn
+            output_board
+            input = get_valid_move(board: game_state.board, presenter: game_io)
+            board = update_board(input: input)
+            update_game_status(board: board)
+            game_state.switch_players if status == :play
         end
         output_board
         output_win if status == :win
         output_tie if status == :tie 
     end
 
-    def take_turn
-        output_board
-        get_valid_move
-        update_board
-        update_game_status
-        game_state.switch_players if status == :play
-    end
-
     private
+    attr_accessor :game_io, :board_presenter
+
     def output_board
-        game_io.output_board(board: game_state.board, board_presenter: board_presenter)
```

Comment:
I like how you've moved the responsibility of outputting the specific game messaging from `ConsoleIO` into this class - now `ConsoleIO` has a clear responsibility